### ISSUE TITLE
discussion: Support creating lakehouse catalog in risingwave.

### ISSUE THREAD

**Issue by liurenjie1024** [risingwavelabs/risingwave#8603]
### Is your feature request related to a problem? Please describe.

Currently, before risingwave can sink to iceberg,  we need to rely on some external system(flink, spark, etc) to create table for us. There are two drawbacks with this problem:

1. It would be quite inconvenient for users to try our solution. For example, if user want just want to store everything in s3, they need to download flink, hadoop, and get familiar with flink SQL to create catalog/tables.
2. DDLs in two different systems is quite an error prone and misleading.

### Describe the solution you'd like

Support creating iceberg catalog in risingwave as following:

```sql
CREATE CATALOG fs_catalog WITH (
  'type'='iceberg',
  'catalog-type'='hadoop',
  'warehouse' = 'file:///Users/renjieliu/Downloads/a'
);

USE fs_catalog;

CREATE DATABASE demo_db;

USE demo_db;

CREATE TABLE demo_table (
  v1 int,
  v2 bigint,
  v3 varchar
);
```

With this enabled, user can ingesting our solution to iceberg in one shot without any other dependencies.

**Comment by StrikeW** [comment]
After DDL is supported, do you plan to support reading and writing to the `demo_table` directly in RisingWave?

**Comment by liurenjie1024** [comment]
> The reason why RisingWave supports Iceberg is to fit into users' existing tech stack where they have used Spark and Iceberg already. Iceberg is not a out-of-the-box system that you can use without any other dependencies. It's still required a separate process to run compaction, serve ad-hoc queries, and to manage catalogs. It will be non-trivial to build a management system for Iceberg. Therefore, from the product's perspective, I would view lakehouse as a completely distinct product line that RisingWave should only provide minimal functions to integrate, instead of combining them together.

Agree. But the goal here is not to build a full managed solution for iceberg, but to make risingwave easier to use and play with. Adding support for dml statements will make integration tests easier(otherwise I need to download other systems such as spark/flink/haddoop to create catalog for us), this will also make beginners who wants to try risingwave with iceberg easier. This is quite similar to other dml statements in our system, they are not intended for production use, but makes tests and playground easier to use.
